src/server.h

```diff
@@ -663,6 +665,10 @@ class Server : public con::PeerHandler, public MapEventReceiver,
 
 	std::unordered_map<std::string, ModMetadata *> m_mod_storages;
 	float m_mod_storage_save_timer = 10.0f;
+
+	// CSM flavour limits byteflag
+	u64 m_csm_flavour_limits = CSMFlavourLimit::CSM_FL_NONE;
+	u32 m_csm_noderange_limit = 8;
```

Comment:
exact but this permit to ensure on different constructor we have a default value set